quantity_warn_on_lot/models/stock_move_split.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+
+from openerp.osv import orm
+from openerp.tools.translate import _
+
+
+class InvalidAmountException(Exception):
+
+    """
+    InvalidAmountException is raised when there is not enough products.
+
+    It contains the product_id, prodlot_id and the actual quantity of
+    products after a successful update.
+    """
+
+    def __init__(self, product_id, prodlot_id, quantity):
+        """
+        Create the InvalidAmountException object.
+
+        :params product_id: The product in question
+        :params prodlot_id: The prodlot in question
+        :params quantity: The quantity of products available
+        """
+        self.product_id = product_id
+        self.prodlot_id = prodlot_id
+        self.quantity = quantity
+
+
+class StockMoveSplit(orm.TransientModel):
```

Comment:
@llacroix In version 7.0 is more common to put wizards (models and views) in the wizard namespace (folder).